pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/ASTParserUtils.groovy

```diff
@@ -422,7 +422,10 @@ class ASTParserUtils {
         Descriptor<? extends Describable> funcDesc = lookupCache.lookupFunction(methodName, descClass)
         StepDescriptor stepDesc = lookupCache.lookupStepDescriptor(methodName)
         // This is the case where we've got a wrapper in options
-        if (stepDesc != null || (funcDesc != null && !StepDescriptor.metaStepsOf(methodName).isEmpty())) {
+        if (stepDesc != null ||
+            (funcDesc != null &&
+                // Deal with cases like jira-steps that claim to be metasteps without any metastep arguments!
+                StepDescriptor.metaStepsOf(methodName)?.first()?.metaStepArgumentType != Object.class)) {
```

Comment:
Should this not be covered by the `workflow-step-api` fix? If not, why not?

Reply:
Yup, I missed a commit.